src/components/Requirements/MultiplePlansDropdown.vue

```diff
@@ -0,0 +1,139 @@
+<template>
+  <div>
+    <div class="multiplePlans-dropdown">
+      <div
+        class="multiplePlans-dropdown-placeholder wrapper"
+        @click="closeDropdownIfOpen()"
+        data-cyid="multiplePlans-wrapper"
+        :class="{ plansHidden: shown }"
+      >
+        <img :src="editplan" class="multiplePlans-dropdown-placeholder editimg" />
+        <div class="multiplePlans-dropdown-placeholder plan">{{ currplan }}</div>
+        <div class="multiplePlans-dropdown-placeholder down-arrow" v-if="!shown"></div>
+        <div class="multiplePlans-dropdown-placeholder up-arrow" v-if="shown"></div>
+      </div>
+      <div class="multiplePlans-dropdown-content" v-if="shown">
+        <div
+          v-for="plan in filterPlans"
+          class="multiplePlans-dropdown-content item"
+          :key="plan"
+          @click="planClicked(plan)"
+        >
+          {{ plan }}
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue';
+import editplan from '@/assets/images/editplan.svg';
+
+export default defineComponent({
+  props: {
+    plan: { type: String, default: 'No additional plans yet' },
+  },
+  data() {
+    return {
+      plans: ['PLAN 1', 'PLAN 2'],
+      shown: false,
+      currplan: 'PLAN 1',
+      editplan,
+    };
+  },
+  computed: {
+    filterPlans() {
+      const filtered = this.$data.plans.filter(plan => plan !== this.$data.currplan);
+      return filtered.length === 0 ? ['No additional plans yet'] : filtered;
+    },
+  },
+  methods: {
+    closeDropdownIfOpen() {
+      this.shown = !this.shown;
+    },
+    planClicked(plan: string) {
+      if (this.plans.length !== 1) this.currplan = plan;
+    },
+  },
+});
+</script>
+
+<style lang="scss">
+@import '@/assets/scss/_variables.scss';
+.multiplePlans-dropdown {
+  display: flex;
+  flex-direction: column;
+
+  &-placeholder {
+    height: 100%;
+    font-size: 16px;
+    font-weight: bold;
+    line-height: 16px;
+    margin-left: 0.25rem;
+    display: flex;
+    align-items: center;
+
+    color: #636363;
```

Comment:
Can we either use a similar color from `src/assets/constants/colors` or add this there if need be?